Amazon.IonDotnet.Tests/Common/ReaderTestCommon.cs

```diff
@@ -168,12 +168,60 @@ public static void ReadAnnotations_SingleField(IIonReader reader)
             Assert.AreEqual("withannot", reader.CurrentFieldName);
             Assert.AreEqual(18, reader.IntValue());
 
+            string[] annotations = reader.GetTypeAnnotations();
+            Assert.AreEqual(symbols.Count(), annotations.Count());
+            Assert.IsTrue(symbols.SequenceEqual(annotations));
+        }
+
+        public static void ReadTypeAnnotationSymbols_SingleField(IIonReader reader)
+        {
+            // a singlefield structure with annotations
+            // {withannot:years::months::days::hours::minutes::seconds::18}
+            var symbols = new[] { "years", "months", "days", "hours", "minutes", "seconds" };
+
+            reader.MoveNext();
+            reader.StepIn();
+            reader.MoveNext();
+            Assert.AreEqual(IonType.Int, reader.CurrentType);
+            Assert.AreEqual("withannot", reader.CurrentFieldName);
+            Assert.AreEqual(18, reader.IntValue());
+
+            Assert.AreEqual(symbols.Count(), reader.GetTypeAnnotationSymbols().Count());
+
             foreach (var s in symbols)
             {
-                Assert.IsTrue(reader.GetTypeAnnotations().Any(a => a.Text == s));
+                Assert.IsTrue(reader.GetTypeAnnotationSymbols().Any(a => a.Text == s));
             }
         }
 
+        public static void HasAnnotationTrue_SingleField(IIonReader reader)
+        {
+            // a singlefield structure with annotations
+            // {withannot:years::months::days::hours::minutes::seconds::18}
+            reader.MoveNext();
+            reader.StepIn();
+            reader.MoveNext();
+            Assert.AreEqual(IonType.Int, reader.CurrentType);
+            Assert.AreEqual("withannot", reader.CurrentFieldName);
+            Assert.AreEqual(18, reader.IntValue());
+
+            Assert.IsTrue(reader.HasAnnotation("days"));
```

Comment:
Might as well check all of the annotations using `HasAnnotation`.

Reply:
Fixed.